tools/chpldoc/chpldoc.cpp

```diff
@@ -393,6 +395,17 @@ static bool isNoWhereDoc(const Function* f) {
   return false;
 }
 
+static bool isHideImplType(const Decl* e) {
+  if (auto attrs = e->attributeGroup()) {
```

Comment:
`ast->hasAttribute()`?

I search and see we don't have it, but I think we should add it now (we already have `hasPragma`).